npm/vue/cypress/integration/spec.js

```diff
@@ -1,3 +1,6 @@
-it('has dummy integration spec', () => {
-  // just to avoid cypress run from failing
+describe('kek', () => {
```

Comment:
Is 'kek' === 'lol'? 👾 

Should probably rename 😛

Reply:
This file useless